todo_app/templates/index.html

```diff
@@ -15,17 +15,36 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
       <ul class="list-group mb-4">
-        {% for item in items %}
-          <li>{{ item.name }} - {{item.status}}</li>
-          {% if item.status != "Complete" %}
+        <h2>To Do Items</h2>
+        {% for item in view_model.items %}  
+          {% if item.status == "To Do" %}
```

Comment:
Here we're missing out on the value of our filtering - if we instead use the `todo_items` property, then we don't need to also explicitly filter here:
```suggestion
        {% for item in view_model.todo_items %}  
```
(Note that we'll need to remove the corresponding `endif`, and we can apply the same idea to the other two statuses)